### PR TITLE
Drop codecov

### PR DESCRIPTION
**Pull request by maarcingebala**
Port #12532 

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Review by jakubkuc — APPROVED** [review]

File: saleor/graphql/product/tests/mutations/test_product_create.py [hunk 1]
```diff
@@ -2156,61 +2156,6 @@ def test_create_product_with_swatch_attribute_new_attribute_value(
...
 
-def test_create_product_with_swatch_attribute_existing_value_using_values_field(
-    staff_api_client,
-    product_type,
-    category,
-    swatch_attribute,
-    permission_manage_products,
-):
-    query = CREATE_PRODUCT_MUTATION
-
```
Comment (NyanKiyoshi) [inline saleor/graphql/product/tests/mutations/test_product_create.py:2167]: Why are we deleting a test case?

**Review by NyanKiyoshi — CHANGES_REQUESTED** [review]

File: saleor/graphql/product/tests/mutations/test_product_create.py (hunk 1 above)
Comment (maarcingebala) [reply saleor/graphql/product/tests/mutations/test_product_create.py:2167]: Because it's duplicated.

Comment (NyanKiyoshi) [reply saleor/graphql/product/tests/mutations/test_product_create.py:2167]: Duplicate of?

Comment (maarcingebala) [reply saleor/graphql/product/tests/mutations/test_product_create.py:2167]: This test is duplicated in this file.

Comment (NyanKiyoshi) [reply saleor/graphql/product/tests/mutations/test_product_create.py:2167]: Which test case name?

Comment (maarcingebala) [reply saleor/graphql/product/tests/mutations/test_product_create.py:2167]: The one we're discussing - `test_create_product_with_swatch_attribute_existing_value_using_values_field`

**Review by NyanKiyoshi — APPROVED** [review]
LGTM

**Pull request merged**